include/bbp/sonata/config.h

```diff
@@ -183,12 +200,6 @@ class SONATA_API CircuitConfig
         std::string biophysicalNeuronModelsDir;
     };
 
-    struct SubnetworkFiles {
```

Comment:
why is this removed?

Reply:
good eye; it wasn't used, and should have been removed before.